pkg/dogecoin/libdogecoin.go

```diff
@@ -0,0 +1,28 @@
+package dogecoin
+
+import (
+	giga "github.com/dogecoinfoundation/gigawallet/pkg"
+
+	"github.com/jaxlotl/go-libdogecoin"
+)
+
+var _ giga.DogecoinL1 = DogecoinL1Libdogecoin{}
```

Comment:
Ok so this is really cool, I had never read about 'interface guards' before, it's a neat idea, love it :)  

I do think because it's a bit non-obvious what it is for (unless you know about interface guards!) it might be worth adding a comment where we do it just so folk are not confused like // Interface Guard ?

Reply:
definitely